### DISCUSSION TITLE
Start Catalyst + Makeswift problem

### DISCUSSION THREAD

**Discussion by anemos474** [bigcommerce/catalyst#1330] [General]
I'm trying to start a Catalyst + Makeswift project. I am currently developing locally.
There were some problems:

1) Makeswift does not allow to make changes. No matter which page I go to, I get the message: "This page is not editable"
2) I get errors in the console when working in makeswift:

`./node_modules/slate-react/dist/index.es.js
Attempted import error: 'useState' is not exported from 'react' (imported as 'useState').

Import trace for requested module:
./node_modules/slate-react/dist/index.es.js
./node_modules/@makeswift/runtime/dist/esm/slate/BuilderPlugin/index.js
./node_modules/@makeswift/runtime/dist/esm/slate/index.js
./node_modules/@makeswift/runtime/dist/esm/controls/rich-text-v2/rich-text-v2.js
./node_modules/@makeswift/runtime/dist/esm/components/builtin/Text/register.js
./node_modules/@makeswift/runtime/dist/esm/components/builtin/register.js
./node_modules/@makeswift/runtime/dist/esm/runtimes/react/react-runtime.js
./node_modules/@makeswift/runtime/dist/esm/runtimes/react/index.js
./node_modules/@makeswift/runtime/dist/esm/react.js
./node_modules/@makeswift/runtime/dist/esm/next/draft-mode/draft-mode-script.js
./node_modules/@makeswift/runtime/dist/esm/next/server.js
./app/[locale]/layout.tsx`

OS Windows 10
node v20.13.1
npm 10.5.2

**Comment by Fefe-Hern** [comment]
For the first issue, does it work if you update the file [...rest]/page.tsx line:

```js
const snapshot = await client.getPageSnapshot(path, {
    siteVersion: getSiteVersion(),
    //locale: params.locale, //Remove this line
  });
```

**Reply by anemos474** [reply]
Thanks for the answer.
Unfortunately, the result did not change.
Still: "This page is not editable"

**Reply by anemos474** [reply]
Also in the browser console in Makeswit tools, received block by CORS :
from origin 'https://app.makeswift.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled
